app.rb

```diff
@@ -61,7 +64,7 @@
       end
     end
 
-    pull_request.check_commits_style if redmine_issue_repos.include?(pull_request.repo) && pr_action != 'created'
+    pull_request.check_commits_style(redmine_project_for_repo(pull_request.repo)) if pr_action != 'created'
```

Comment:
a) check_commits_style doesn't take an argument
b) this removes the check for projects that have the check enabled